app/build.gradle

```diff
@@ -100,4 +100,14 @@ dependencies {
     exclude group: 'commons-logging', module: 'commons-logging'
     exclude group: 'org.apache.httpcomponents', module: 'httpclient'
   }
+  compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
Duplicate dependency on line #76. Let's delete that line in favor of this one that uses the `$kotlin_version` variable.